diff --git a/common/application.cpp b/common/application.cpp
index afdd08b..79c4cb0 100644
--- a/common/application.cpp
+++ b/common/application.cpp
@@ -372,22 +372,22 @@ void do_data_dir(std::filesystem::path dir, std::unordered_map<std::string, appl
 
 std::unordered_map<std::string, application> list_applications()
 {
-	std::unordered_map<std::string, application> res;
+	// std::unordered_map<std::string, application> res;
+// 
+	// read_steam_vr_apps(res);
+// 
+	// do_data_dir(xdg_data_home(), res);
+// 
+	// for (auto && dir: xdg_data_dirs())
+	// 	do_data_dir(std::move(dir), res);
+// 
+	// if (wivrn::is_flatpak())
+	// {
+	// 	// Try to guess host data dirs
+	// 	do_data_dir("/run/host/usr/share", res);
+	// }
 
-	read_steam_vr_apps(res);
-
-	do_data_dir(xdg_data_home(), res);
-
-	for (auto && dir: xdg_data_dirs())
-		do_data_dir(std::move(dir), res);
-
-	if (wivrn::is_flatpak())
-	{
-		// Try to guess host data dirs
-		do_data_dir("/run/host/usr/share", res);
-	}
-
-	return res;
+	return {};
 }
 
 } // namespace wivrn
diff --git a/fix.patch b/fix.patch
new file mode 100644
index 0000000..e69de29
